projects/storefrontstyles/scss/components/form-components/_star-rating.scss

```diff
@@ -1,19 +1,26 @@
 %cx-star-rating {
-  .star {
-    position: var(--cx-position, relative);
-    display: var(--cx-display, inline-block);
-    color: var(--cx-color, var(--cx-g-color-light));
-    margin: var(--cx-margin, 0 5px 0 0);
-  }
+  padding: 15px 0;
 
-  .full {
-    color: var(--cx-color, var(--cx-g-color-primary));
+  &:not([disabled='true']) {
+    cursor: pointer;
   }
 
-  .half {
-    position: var(--cx-position, absolute);
-    display: var(--cx-display, inline-block);
-    overflow: var(--cx-overflow, hidden);
-    color: var(--cx-color, var(--cx-g-color-primary));
+  .star {
+    padding-right: 5px;
+    font-size: 20px;
+
+    @for $i from 1 to 6 {
+      &:nth-child(#{$i}) {
+        background: linear-gradient(
+          90deg,
+          var(--cx-g-color-primary) calc((var(--star-fill) - #{$i} + 1) * 100%),
```

Comment:
Not sure about the fill calculation.

Before: 
<img width="451" alt="Screen Shot 2019-05-03 at 9 00 40 PM" src="https://user-images.githubusercontent.com/10605724/57171902-2eb50900-6de7-11e9-9f35-520a9c65dc84.png">

After:
<img width="456" alt="Screen Shot 2019-05-03 at 8 59 53 PM" src="https://user-images.githubusercontent.com/10605724/57171922-79cf1c00-6de7-11e9-9de9-3e50a0c3388a.png">

Supposed to be 2.6:
<img width="693" alt="Screen Shot 2019-05-03 at 9 00 55 PM" src="https://user-images.githubusercontent.com/10605724/57171919-6de35a00-6de7-11e9-93a1-e8c53f8874ad.png">

I feel like the before is closer to 2.6 than the after, the after looks like 2.8-2.9.

Reply:
Good spot! I've fixed it by changing the padding into margin.